File: other/fizzbuzz/fizzbuzz.java
public class fizzbuzz {
	public static void main(String[] args) {
		fizzbuzzToVal(100);	
	}
	
	private static void fizzbuzzToVal(int val){
		String outStr;
		for (int i = 0; i <= val; i++){
			outStr = "";
			if (i % 3 == 0){
				outStr += "Fizz";
			}
			if (i % 5 == 0){
				outStr += "Buzz";
			}
			if (outStr.isEmpty()){
				outStr = String.valueOf(i);
			}
			System.out.println(outStr);
		}
	}
}

File: Python/FizzBuzz/fizzbuzz.py
def doFizzBuzz(numTo):
    for i in range(1,numTo+1):
        if i % 3 == 0:
            print("Fizz", end = '')
        if i % 5 == 0:
            print("Buzz", end = '')
        if( i % 3 != 0 and i % 5 != 0):
            print(i, end = '')
        print()
        

doFizzBuzz(50)
